### PR TITLE
docs(misc): community slack to official discord

### PR DESCRIPTION
**Pull request by ZackDeRose**
## Current Behavior

Points people to the community slack (most of the links are now redirected to discord, but looks like a couple still go straight to the community slack)

## Expected Behavior
Points people to the official discord server.

## Related Issue(s)

Fixes #

### CODE REVIEW

File: docs/shared/guides/turbo-and-nx.md [hunk 1]
```diff
@@ -173,6 +173,5 @@ Nx was released in 2016. Turborepo was open sourced in December of 2021. Turbore
 - There are about [3 million downloads per week](https://www.npmjs.com/package/@nx/tao).
 - There are about 1 million+ unique [Nx Console](https://marketplace.visualstudio.com/items?itemName=nrwl.angular-console) (a plugin for VSCode) installations.
 - There is a rich ecosystem of [third-party plugins.](https://nx.dev/community)
-- There are 5500+ active users in [the community slack.](https://nrwlcommunity.slack.com/join/shared_invite/zt-jba969hz-e5zm1Ou_jiOP1J2h8UTu0w#/shared-invite/email)
```
Comment (ZackDeRose) [inline docs/shared/guides/turbo-and-nx.md:176]: I removed this as our numbers now at not very impressive given we just launched today (~150 on day one it looks like)

**Review by Moxita33 — APPROVED** [review]

File: docs/shared/react-standalone-tutorial/react-standalone.md [hunk 2]
```diff
@@ -834,7 +834,7 @@ Here's some more things you can dive into next:
 
 Also, make sure you
 
-- [Join the Nx community Slack](https://go.nrwl.io/join-slack) to ask questions and find out the latest news about Nx.
+- [Join the Official Nx Server](http://go.nx.dev/community) to ask questions and find out the latest news about Nx.
```
Comment (isaacplmann) [inline docs/shared/react-standalone-tutorial/react-standalone.md:837]:
```suggestion
- [Join the Official Nx Discord Server](http://go.nx.dev/community) to ask questions and find out the latest news about Nx.
```

**Review by isaacplmann — CHANGES_REQUESTED** [review]
Found a typo

**Review by juristr — APPROVED** [review]

**Review by isaacplmann — APPROVED** [review]

File: scripts/readme-fragments/links.md [hunk 3]
```diff
@@ -6,6 +6,6 @@
 [![Semantic Release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg?style=flat-square)]()
 [![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
 [![Join the chat at https://gitter.im/nrwl-nx/community](https://badges.gitter.im/nrwl-nx/community.svg)](https://gitter.im/nrwl-nx/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[![Join us @nx/community on slack](https://img.shields.io/badge/slack-%40nrwl%2Fcommunity-brightgreen)](https://join.slack.com/t/nrwlcommunity/shared_invite/enQtNzU5MTE4OTQwOTk0LTgxY2E0ZWYzMWE0YzA5ZDA2MWM1NDVhNmI2ZWMyYmZhNWJiODk3MjkxZjY3MzU5ZjRmM2NmNWU1OTgyZmE4Mzc)
+[![Join us on the Official Nx Discord Server](https://simpleicons.org/icons/discord.svg)](http://go.nx.dev/community)
```
Comment (FrozenPandaz) [inline scripts/readme-fragments/links.md:9]: The link here should be `https`

File: README.md [hunk 4]
```diff
@@ -13,7 +13,7 @@
 [![Semantic Release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg?style=flat-square)]()
 [![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
 [![Join the chat at https://gitter.im/nrwl-nx/community](https://badges.gitter.im/nrwl-nx/community.svg)](https://gitter.im/nrwl-nx/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[![Join us @nrwl/community on slack](https://img.shields.io/badge/slack-%40nrwl%2Fcommunity-brightgreen)](https://go.nrwl.io/join-slack)
+[![Join the Official Nx Discord Server](https://img.shields.io/discord/1143497901675401286?label=discord)](http://go.nx.dev/community)
```
Comment (FrozenPandaz) [inline README.md:16]: The link here should be `https`

File: .github/ISSUE_TEMPLATE/config.yml [hunk 5]
```diff
@@ -7,8 +7,8 @@ contact_links:
     about: "Want to start a thread to discuss an idea? Use the discussions feature provided by GitHub."
     url: https://github.com/nrwl/nx/discussions
   - name: Have a question?
-    url: https://go.nrwl.io/join-slack
-    about: "The Community Slack is a great place for questions to be asked and answered. Please use the #support channel if you need help with your workspace!"
+    url: http://go.nx.dev/community
```
Comment (FrozenPandaz) [inline .github/ISSUE_TEMPLATE/config.yml:10]: The link here should be `https`

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]

**Comment by ZackDeRose** [comment]
Thanks @FrozenPandaz!! Updated for https

**Review by jaysoo — APPROVED** [review]

File: docs/shared/react-standalone-tutorial/react-standalone.md [hunk 6]
```diff
@@ -834,7 +834,7 @@ Here's some more things you can dive into next:
 
 Also, make sure you
 
-- [Join the Nx community Slack](https://go.nrwl.io/join-slack) to ask questions and find out the latest news about Nx.
+- [Join the Official Nx Discord Server](http://go.nx.dev/community) to ask questions and find out the latest news about Nx.
```
Comment (FrozenPandaz) [inline docs/shared/react-standalone-tutorial/react-standalone.md:837]: Missed one.

Could you open a separate PR to change all the `http://` in here please?

**Review by FrozenPandaz — APPROVED** [review]

**Pull request merged**